app/src/main/AndroidManifest.xml

```diff
@@ -34,6 +34,9 @@
             android:label="@string/app_name"
             android:theme="@style/AppTheme.NoActionBar">
         </activity>
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="637dc807ac8096c2df2f153c1fb89fa7a61ee415" />
```

Comment:
Is this safe to expose on a public repo? Why do we need both a fabric api key and a firebase api key?

Reply:
https://github.com/zooniverse/vox/issues/32

Reply:
Also Firebase is not just about Crash analytics, it comes with many other features like general usage patterns, auth and much more. Now firebase itself uses fabric for providing crash analytics and makes you register separately for crash analytics with fabric.
Though after some time both will be merged together but still API key will be required for both.

Reply:
I saw the zooniverse link, but that seemed to refer to the firebase api key, not the fabric api key. Most sources I could find regarding the fabric api key are implying that it's not safe to release, but I know the recent project merger may be changing things.

Reply:
We can go for something like this 
https://github.com/codepath/android_guides/wiki/Storing-Secret-Keys-in-Android